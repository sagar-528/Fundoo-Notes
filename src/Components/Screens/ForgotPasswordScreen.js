import React from 'react'
import ForgetPassword from '../ForgotPassword'

const ForgotPasswordScreen = ( {navigation} ) => {
    return (
        <ForgetPassword navigation={navigation} />
    )
}

export default ForgotPasswordScreen
