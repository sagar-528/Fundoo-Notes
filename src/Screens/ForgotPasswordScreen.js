import React from 'react'
import ForgetPassword from '../Components/ForgotPassword'

const ForgotPasswordScreen = ( {navigation} ) => {
    return (
        <ForgetPassword navigation={navigation} />
    )
}

export default ForgotPasswordScreen
