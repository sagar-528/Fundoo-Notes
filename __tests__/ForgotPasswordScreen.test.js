import React from 'react';
import {shallow} from 'enzyme';
import ForgotPasswordScreen from '../src/Screens/ForgotPasswordScreen'

describe('test ForgotPasswordScreen', () => {
    it('Should match to snapshot', () => {
        const component = shallow(<ForgotPasswordScreen/>)
        expect(component).toMatchSnapshot();
    })
})