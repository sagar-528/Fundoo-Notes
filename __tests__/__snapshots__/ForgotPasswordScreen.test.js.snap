// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test ForgotPasswordScreen should match to snapshot 1`] = `ShallowWrapper {}`;
